>From f4ac11faf518c61e5cb7a0ef4fb19bd3348e7646 Mon Sep 17 00:00:00 2001
From: Florian Weimer <fweimer@redhat.com>
Date: Tue, 10 Jul 2012 13:41:34 +0200
Subject: [PATCH 5/6] src/condor_startd.V6/startd_cron_job_params.h: remove
 using namespace std

---
 src/condor_startd.V6/ResAttributes.cpp             |    2 +-
 src/condor_startd.V6/Starter.cpp                   |    2 +-
 src/condor_startd.V6/startd_cron_job_params.h      |    3 +--
 src/condor_startd.V6/startd_named_classad_list.cpp |    2 +-
 4 files changed, 4 insertions(+), 5 deletions(-)

diff --git a/src/condor_startd.V6/ResAttributes.cpp b/src/condor_startd.V6/ResAttributes.cpp
index 8bbe049..0c045d7 100644
--- a/src/condor_startd.V6/ResAttributes.cpp
+++ b/src/condor_startd.V6/ResAttributes.cpp
@@ -552,7 +552,7 @@ void MachAttributes::init_machine_resources() {
         const string invprefix = "INVENTORY_";
         const string restr = prefix + "(.+)";
         ASSERT(re.compile(restr.c_str(), &pszMsg, &err, PCRE_CASELESS));
-        vector<string> resdef;
+	std::vector<std::string> resdef;
         const int n = param_names_matching(re, resdef);
         for (int j = 0;  j < n;  ++j) {
             string rname = resdef[j].substr(prefix.length());
diff --git a/src/condor_startd.V6/Starter.cpp b/src/condor_startd.V6/Starter.cpp
index 921c0ae..5e2cfbb 100644
--- a/src/condor_startd.V6/Starter.cpp
+++ b/src/condor_startd.V6/Starter.cpp
@@ -1320,7 +1320,7 @@ Starter::startKillTimer( void )
 		// we keep trying.
 	s_kill_tid = 
 		daemonCore->Register_Timer( tmp_killing_timeout,
-									max(1,tmp_killing_timeout),
+									std::max(1,tmp_killing_timeout),
 						(TimerHandlercpp)&Starter::sigkillStarter,
 						"sigkillStarter", this );
 	if( s_kill_tid < 0 ) {
diff --git a/src/condor_startd.V6/startd_cron_job_params.h b/src/condor_startd.V6/startd_cron_job_params.h
index 6eb0575..cb6b11c 100644
--- a/src/condor_startd.V6/startd_cron_job_params.h
+++ b/src/condor_startd.V6/startd_cron_job_params.h
@@ -22,7 +22,6 @@
 
 #include "classad_cron_job.h"
 #include <list>
-using namespace std;
 
 // Define a "ClassAd" cron job parameter object
 class StartdCronJobParams : public ClassAdCronJobParams
@@ -37,7 +36,7 @@ class StartdCronJobParams : public ClassAdCronJobParams
 	bool InSlotList( unsigned slot ) const;
 
   private:
-	list<unsigned>	m_slots;
+	std::list<unsigned>	m_slots;
 };
 
 #endif /* _STARTD_CRON_JOB_PARAMS_H */
diff --git a/src/condor_startd.V6/startd_named_classad_list.cpp b/src/condor_startd.V6/startd_named_classad_list.cpp
index 070d937..6b105e1 100644
--- a/src/condor_startd.V6/startd_named_classad_list.cpp
+++ b/src/condor_startd.V6/startd_named_classad_list.cpp
@@ -42,7 +42,7 @@ StartdNamedClassAdList::Register( StartdNamedClassAd *ad )
 int
 StartdNamedClassAdList::Publish( ClassAd *merged_ad, unsigned r_id )
 {
-	list<NamedClassAd *>::iterator iter;
+	std::list<NamedClassAd *>::iterator iter;
 	for( iter = m_ads.begin(); iter != m_ads.end(); iter++ ) {
 		NamedClassAd		*nad = *iter;
 		StartdNamedClassAd	*sad = dynamic_cast<StartdNamedClassAd*>(nad);
-- 
1.7.7.6

